resources/schema.py

```diff
@@ -41,6 +42,40 @@ def update_object(obj, data):
     obj.save()
 
 
+def _resolve_piers(**kwargs):
+    min_width = kwargs.get("min_berth_width", 0)
+    min_length = kwargs.get("min_berth_length", 0)
+    filter_empty_berths = any(
```

Comment:
could you put a comment here: "Filter out piers with no berths, that fit passed dimensions, only if the dimensions were given"?

Reply:
afraid it might be a bit confusing to see this without at least some context